assets/js/translator.js

```diff
@@ -447,12 +447,23 @@ function getPairs() /*: JQueryPromise<any> */ {
             }
         });
 
-        for(var srcLang in pairs) {
-            // Default for new users is first available pair
-            curSrcLang = srcLang;
-            curDstLang = pairs[srcLang][0];
-            break;
+        // Default for new users is first available Browser Preference Language pair
+        var prefLang;
+        if(languages[navigator.languages[0]] !== undefined) {
```

Comment:
What is the availability of the `navigator.languages` API? html-tools currently maintains compatibility with a relatively old set of browsers.

Secondly, why is this not possible with a loop...?

Reply:
It is available for all Chrome, Safari and Opera Versons and Firefox 1.0

Reply:
Yeahh Its possible. :).

Reply:
Loop.. Done. :)

Reply:
What about IE though...?

Reply:
IE??

Reply:
Internet Explorer, 11.0

Reply:
Okay.. Now it will support IE also. :)